packages/react-search-ui-views/src/SingleSelectFacet.js

```diff
@@ -19,27 +21,14 @@ Option.propTypes = {
   data: PropTypes.object.isRequired
 };
 
-function toValue(from, to) {
```

Comment:
We used to do some funky stringification of values for the `Select` box. The logic was specific to Range type filters and as it turns out, was 100% unnecessary. We now just pass the value as the `value` to the `Select` and don't worry about what type of value it is.